src/index.html

```diff
@@ -16,17 +19,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__body">
+    <form
+      class="search-bar__form search-bar__form--big"
+      action="http://localhost:8080/"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input search-bar__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        autocomplete="on"
```

Comment:
The `input` tag's attributes should follow the same indentation rules as above. Every attribute should be on a new line with 2-space indentation.